src/transformState.js

```diff
@@ -45,8 +45,47 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      addProperties(state, transforms[i].properties);
+    }
+
+    if (transforms[i].operation === 'removeProperties') {
+      removeProperties(state, transforms[i].properties);
+    }
+
+    if (transforms[i].operation === 'clear') {
+      clear(state, transforms[i].properties);
+    }
```

Comment:
consider of using switch construction or if => else (there is no way that you received two different operations, so no need letting your program acting in this way)